test/has-route.test.js

```diff
@@ -71,7 +71,23 @@ test('hasRoute', t => {
 
     t.equal(fastify.hasRoute({
       method: 'GET',
-      url: '/example/12345.png'
+      url: '/example/:file(^\\d+).png'
+    }), true)
+  })
+
+  test('hasRoute - finds a route even if method is not uppercased', t => {
+    t.plan(1)
+    fastify.route({
+      method: 'GET',
+      url: '/equal',
+      handler: function (req, reply) {
+        reply.send({ hello: 'world' })
+      }
+    })
+
+    t.equal(fastify.hasRoute({
+      method: 'get',
+      url: '/equal'
```

Comment:
The options are wrong, you need to pass an `options` object

Reply:
?

Reply:
i think this is correct with the new hasRoute method

Reply:
I oversee I think, so far the coverage is complaining; sorry, my wires mixed 👍